app/templates/meetings/edit/outro.hbs

```diff
@@ -0,0 +1,29 @@
+<AuModal @modalOpen={{true}}
+         @size="fullscreen"
+         @modalTitle={{t "meetingForm.sixthSectionTitle"}}
+         @closeModal={{this.closeModal}}
+         as |Modal|>
+  <Modal.Body>
+    <ZittingTextDocumentContainer
+            @onEditorInit={{this.initEditor}}
+            @saving={{this.saveTextTask.isRunning}}
+            @type="ext:outro"
+            @zitting={{@model}}/>
+  </Modal.Body>
+  <Modal.Footer>
+    <AuButtonGroup>
+      <AuButton @skin="secondary"
```

Comment:
personally, I'd just disable the buttons when saveTextTask is running and include a spinner next to the buttons.

Reply:
I don't even think we need the spinner, we already have the scanner animation